docs/root/version_history/current.rst

```diff
@@ -17,6 +17,9 @@ Changes
   Can be reverted temporarily by setting runtime feature `envoy.reloadable_features.fix_upgrade_response` to false.
 * http: remove legacy connection pool code and their runtime features: `envoy.reloadable_features.new_http1_connection_pool_behavior` and
   `envoy.reloadable_features.new_http2_connection_pool_behavior`.
+* listener: added additional draining filter chain stat for :ref:`listener manager <config_listener_manager_stats>`.
```

Comment:
I would remove this release note or merge it into the next one if you wish.

Reply:
Will do